### ISSUE TITLE
Unit test error BCAD v24, UnitTestPyBcadCivil

### ISSUE THREAD

**Issue by schoeller** [CEXT-Dan/PyRx#9]
Dear all,

thanks for the courageous work. 
When running the unit test on BCAD I receive the following error.

```python
<unittest.runner.TextTestResult run=1 errors=0 failures=0>
TestBricsCADCivil
======================================================================
FAIL: test_BsysCvDbAlignment_cast (UnitTestPyBcadCivil.TestBCadCivil.test_BsysCvDbAlignment_cast)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\temp/PyRx-1.2.1/PyRx-1.2.1/unit tests\UnitTestPyBcadCivil.py", line 56, in test_BsysCvDbAlignment_cast
    self.assertNotEqual(line.length(), 0)
AssertionError: 0.0 == 0

```
Best regards

Sebastian

**Comment by CEXT-Dan** [comment]
Hi Sebastian,

This is passing for me, but I see that the test has issues, I’ll be working on this 

![test](https://github.com/CEXT-Dan/PyRx/assets/26830697/481f6f1f-f0d6-4b41-a8d1-966452997453)

**Comment by CEXT-Dan** [comment]
Hi, can you re check this? I've modified the test, it should test of the length is greater than zero, in case it was a tolerance issue

**Comment by schoeller** [comment]
Pulled from origin PyRx version <1.2.007.20240329>. Unittest is not working. Handle `AE` exist. To check on length I modified test to 

```python
        for id in hAlignment.getUnorderedElementIds():
            element : Cv.CvDbHAlignmentElement = hAlignment.elementAtId(id)
            self.assertFalse(element.isNullObj())
            if Cv.HAlignmentElementType.eLine == element.type():
                line: Cv.CvDbHAlignmentLine = Cv.CvDbHAlignmentLine.cast(element)
                print(line.length())
                # self.assertGreater(line.length(), 0)
                # flag = line.isA().isDerivedFrom(Cv.CvDbHAlignmentLine.desc())
                # self.assertTrue(flag)
            elif Cv.HAlignmentElementType.eSpiralCurveSpiral == element.type():
                spiral : Cv.CvDbHAlignmentSCS = Cv.CvDbHAlignmentSCS.cast(element)
                print(spiral.length())
                # self.assertGreater(spiral.length(), 0)
                # flag = spiral.isA().isDerivedFrom(Cv.CvDbHAlignmentSCS.desc())
                # self.assertTrue(flag)
```

Results as follows.

BCAD 24.1.08 German :
```python
: PYBCCIVILTEST
TestBricsCADCivil
0.0
0.0
0.0
----------------------------------------------------------------------
Ran 4 tests in 0.029s

OK
```

BCAD 24.2.03, English
```python
: PYBCCIVILTEST
TestBricsCADCivil
8.07413485431498
5.000049808605995
10.000049808605997
----------------------------------------------------------------------
Ran 4 tests in 0.025s

OK
```

Will try next BCAD 24.1.8 English version.

**Comment by CEXT-Dan** [comment]
Oh, if that’s the case it’s a bug in BricsCAD!

**Comment by schoeller** [comment]
I have create a report with SR175196.

**Comment by schoeller** [comment]
I confirm it has been fixed in BCAD 24.2.03

<img src="https://github.com/CEXT-Dan/PyRx/assets/106036/6ef0459c-2f62-467c-9743-d84b0c53b72d" width="480" />

All test pass

<img src="https://github.com/CEXT-Dan/PyRx/assets/106036/7a288791-36da-457f-ae65-0ef4e53df999" width="480" />
